src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1881,8 +1881,8 @@ impl Cage {
     }
 }
 
-pub fn kernel_close(_fdentry: fdtables::FDTableEntry, kernelfd: u64) {
+pub fn kernel_close(fdentry: fdtables::FDTableEntry, _count: u64) {
```

Comment:
Changing the var name from `kernelfd` to `_count` might be confusing for future devs

Reply:
> The updated fdtable API passing fd entry and count instead of real fd into kernel_close function in RawPOSIX

^ reason of changing the var's name